Assets/Scripts/Enemies/AIMovement.cs

```diff
@@ -49,20 +116,29 @@ void FixedUpdate()
             currentDirection = 1f;
         }
         //Store user input as a movement vector
```

Comment:
Nit: this is not user input, right? Can we remove this comment to avoid confusion?

Reply:
Sure